pkg/config/context.go

```diff
@@ -47,7 +47,7 @@ type Management struct {
 	ControllerFactory controller.SharedControllerFactory
 
 	VirtFactory    *kubevirt.Factory
-	VMFactory      *vm.Factory
+	HarvFactory    *harvester.Factory
```

Comment:
I would prefer HarvesterFactory for the field

Reply:
Done